src/nvim/option.c

```diff
@@ -238,6 +238,8 @@ typedef struct vimoption {
                                    ///< when there is a redraw flag
 #define P_NO_DEF_EXP   0x8000000U  ///< Do not expand default value.
 
+#define P_RWINONLY     0x10000000U  ///< only redraw current window
+
```

Comment:
@justinmk 
Good catch.

Does this patch look good?

```diff
diff --git i/src/nvim/option.c w/src/nvim/option.c
index 76667fa9..a4e7da77 100644
--- i/src/nvim/option.c
+++ w/src/nvim/option.c
@@ -213,12 +213,12 @@ typedef struct vimoption {
 #define P_VI_DEF        0x400U   /* Use Vi default for Vim */
 #define P_VIM           0x800U   /* Vim option */
 
-/* when option changed, what to display: */
-#define P_RSTAT         0x1000U  /* redraw status lines */
-#define P_RWIN          0x2000U  /* redraw current window */
-#define P_RBUF          0x4000U  /* redraw current buffer */
-#define P_RALL          0x6000U  /* redraw all windows */
-#define P_RCLR          0x7000U  /* clear and redraw all */
+// when option changed, what to display:
+#define P_RSTAT         0x1000U  ///< redraw status lines
+#define P_RWIN          0x2000U  ///< redraw current window and recompute text
+#define P_RBUF          0x4000U  ///< redraw current buffer and recompute text
+#define P_RALL          0x6000U  ///< redraw all windows
+#define P_RCLR          0x7000U  ///< clear and redraw all
 
 #define P_COMMA         0x8000U    ///< comma separated list
 #define P_ONECOMMA      0x18000U   ///< P_COMMA and cannot have two consecutive
```

What is the purpose/meaning of `///<` vs just `//`?